src/main/java/org/springframework/data/redis/connection/lettuce/LettuceConverters.java

```diff
@@ -535,6 +566,86 @@ public static RedisURI sentinelConfigurationToRedisURI(RedisSentinelConfiguratio
 		return builder.build();
 	}
 
+	/**
+	 * Converts a {@link RedisURI} to its corresponding {@link RedisSentinelConfiguration}.
+	 *
+	 * @param redisURI the uri containing the Redis Sentinel connection info
+	 * @return a {@link RedisSentinelConfiguration} representing the Redis Sentinel information in the Redis URI.
+	 * @since 2.6
+	 */
+	public static RedisSentinelConfiguration redisUriToSentinelConfiguration(RedisURI redisURI) {
```

Comment:
Let's keep these methods package-private. We don't want to expose these any further.

Reply:
Yes, please. I was not sure if the pattern was to keep these public as `sentinelConfigurationToRedisURI` is public and should have called that out w/ a comment. I too am a member of the "principle of least privilege" camp.